### ISSUE TITLE
Unhandled exception in dns-sync.js when `debug` environment variable set to *

### ISSUE THREAD

**Issue by markSmurphy** [skoranga/node-dns-sync#6]
I have a package which is dependent on dns-sync and which also uses the `debug` package.  I've noticed that if I set `debug=*` and my code performs a DNS lookup via dns-sync I get the error:
```text
SyntaxError: Unexpected token T in JSON at position 0
    at JSON.parse (<anonymous>)
    at Object.resolve (c:\Users\...\staging\node_modules\dns-sync\lib\dns-sync.js:38:25)
    at QueryReqWrap.callback (c:\Users\...\staging\akamai-staging.js:62:38)
    at QueryReqWrap.onresolve [as oncomplete] (dns.js:205:10)
```
Which looks to be because `response.output` isn't valid JSON when debug is enabled, and it trips up at line 38:
>return JSON.parse(response.output);

**Comment by skoranga** [comment]
@markSmurphy let me check.

**Comment by skoranga** [comment]
published `dns-sync@0.2.0` with the fix.
